stix_shifter_modules/aws_athena/stix_translation/json/ocsf_from_stix_map.json

```diff
@@ -6,6 +6,91 @@
       ]
     }
   },
+  "file": {
+    "fields": {
+      "accessed": [
+        "file.accessed_time"
+      ],
+      "created": [
+        "file.created_time"
+      ],
+      "extensions.'x-ocsf-file-ext'.algorithm": [
+        "file.fingerprints.algorithm"
+      ],
+      "extensions.'x-ocsf-file-ext'.algorithm_id": [
+        "file.fingerprints.algorithm_id"
+      ],
+      "extensions.'x-ocsf-file-ext'.algorithm_value": [
+        "file.fingerprints.value"
+      ],
+      "extensions.'x-ocsf-file-ext'.attributes": [
+        "file.attributes"
+      ],
+      "extensions.'x-ocsf-file-ext'.company_name": [
+        "file.company_name"
+      ],
+      "extensions.'x-ocsf-file-ext'.confidentiality": [
+        "file.confidentiality"
+      ],
+      "extensions.'x-ocsf-file-ext'.confidentiality_id": [
+        "file.confidentiality_id"
+      ],
+      "extensions.'x-ocsf-file-ext'.description": [
+        "file.desc"
+      ],
+      "extensions.'x-ocsf-file-ext'.feature_name": [
+        "file.product.feature.name"
+      ],
+      "extensions.'x-ocsf-file-ext'.feature_uid": [
+        "file.product.feature.uid"
+      ],
+      "extensions.'x-ocsf-file-ext'.feature_version": [
+        "file.product.feature.version"
+      ],
+      "extensions.'x-ocsf-file-ext'.installed_path": [
+        "file.product.path"
+      ],
+      "extensions.'x-ocsf-file-ext'.product_lang": [
+        "file.product.lang"
+      ],
+      "extensions.'x-ocsf-file-ext'.product_name": [
```

Comment:
I'm wondering if product name, vendor, and version should all be mapped to the software object instead.

Reply:
you are right. updating